experimenter/experimenter/experiments/models.py

```diff
@@ -643,6 +646,10 @@ def treatment_branches(self):
             branches = branches.exclude(id=self.reference_branch.id)
         return list(branches)
 
+    @property
+    def is_desktop(self):
```

Comment:
Maybe this should be `application_is_desktop` or similar

Reply:
Oh, could be, usually you start predicates like this with the verb so `is_` `can_` `should_` etc, so it'd be more like `is_application_desktop` but I feel like `desktop` is only used in the context of the application so it seems unambiguous to me?